featuretools/tests/utils_tests/test_entity_utils.py

```diff
@@ -17,19 +16,14 @@
 
 
 @pytest.fixture
-def pd_mock_customer_es():
-    return ft.demo.load_mock_customer(return_entityset=True, random_seed=0)
-
-
-@pytest.fixture
-def dask_mock_customer_es(pd_mock_customer_es):
-    dask_es = copy.deepcopy(pd_mock_customer_es)
+def dask_mock_customer_es(pd_mock_customer):
+    dask_es = copy.deepcopy(pd_mock_customer)
     for entity in dask_es.entities:
         entity.df = dd.from_pandas(entity.df.reset_index(drop=True), npartitions=2)
     return dask_es
 
 
-@pytest.fixture(params=['pd_mock_customer_es', 'dask_mock_customer_es'])
+@pytest.fixture(params=['pd_mock_customer', 'dask_mock_customer_es'])
```

Comment:
We can delete this fixture and use the `mock_customer` fixture from `conftest.py` instead. Will also need to update `test_get_linked_vars` to use the `mock_customer` feature as well to test with both pandas and Dask.

Reply:
so there should be 2 more `test_get_linked_vars`, such as `test_get_linked_vars_pd` and `test_get_linked vars_dd`?

Reply:
No, you do not need to create any additional tests. Because `mock_customer` is a parameterized fixture, anywhere it is used, pytest will automatically test using the defined parameter values. So in the current `test_get_linked_vars` if you just use `mock_customer` as the fixture instead, pytest will execute that test with `pd_mock_customer` and then `dd_mock_customer`. The change you have made to this test already should be sufficient.